<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasteRecordDialog</class>
 <widget class="QDialog" name="PasteRecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>790</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>790</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>随手记--流水</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#PasteRecordDialog{border-image: url(:/new/prefix1/image/背景2.PNG);}
</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;华文隶书&quot;;</string>
   </property>
   <property name="text">
    <string>实用报表</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>371</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>371</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>371</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#graphicsView{border-image: url(:/new/prefix1/image/背景1.PNG);}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;方正姚体&quot;;</string>
   </property>
   <property name="text">
    <string>随手一记</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>350</y>
     <width>142</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_17">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QPushButton" name="ButtonFunctionOldthing">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/new/prefix1/image/0.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;方正姚体&quot;;color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>流水</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <widget class="QPushButton" name="ButtonFunctionMessage">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/new/prefix1/image/2.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton7">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;方正姚体&quot;;color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>消息</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_18">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/new/prefix1/image/1.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonFunctionOldThing">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;方正姚体&quot;;color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>记录</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <widget class="QPushButton" name="ButtonExit">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/new/prefix1/image/3.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;方正姚体&quot;;color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>退出</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>280</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/myapp.ico);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;方正姚体&quot;;</string>
   </property>
   <property name="text">
    <string>你就知道</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>310</y>
     <width>177</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_22">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;方正姚体&quot;;color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>功能大全</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonMoreFunction">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>&gt;&gt;&gt;</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ButtonNewAccount">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>230</y>
     <width>160</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/返回主界面.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <row>
    <property name="text">
     <string>支出</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>收入</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>一月</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>二月</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>三月</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>四月</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string>0</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>支出</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>收入</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>五月</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>六月</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>七月</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>八月</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string>777777</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tableWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>支出</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>收入</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>九月</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>十月</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>十一月</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>十二月</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>70</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>70</string>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>90</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>890</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>70</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>7907</string>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string>7</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>144</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>144</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/按日查询.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>450</y>
     <width>144</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>144</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/支出分类排行.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>144</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>144</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/本年收入图表.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>450</y>
     <width>144</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>144</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/本年支出图表.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>144</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>144</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/收入分类排行.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>0</y>
     <width>180</width>
     <height>144</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>144</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>怠惰
是贫穷的制造厂。</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>label_2</zorder>
  <zorder>label_8</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>line_8</zorder>
  <zorder>label_7</zorder>
  <zorder>label_9</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>ButtonNewAccount</zorder>
  <zorder>tableWidget</zorder>
  <zorder>tableWidget_2</zorder>
  <zorder>tableWidget_3</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
